.github/workflows/tests.yml

```diff
@@ -38,10 +38,6 @@ jobs:
           cp tests/acceptance.conf.php.default tests/acceptance.conf.php
           cp tests/smoke.conf.php.default tests/smoke.conf.php
 
-      - name: Require Symfony PHPUnit Bridge 5.4 for PHP 8.1
```

Comment:
A lower version of symfony-bridge was required for PHP < 7.1.3 but as we've removed support for everything < 7.4, this is not required anymore.